packages/appstore/src/constants/tour-steps-config.ts

```diff
@@ -0,0 +1,57 @@
+import { Step, Styles, Locale } from 'react-joyride';
+
+export const tour_step_config: Step[] = [
+    {
+        title: 'Switch accounts',
+        content: 'You can switch between real and demo accounts.',
+        target: '.toggle-account-type__button',
+        disableBeacon: true,
+        placement: 'bottom-end',
+    },
+    {
+        title: 'Trading Hub tour',
+        content: `Need help moving around?\n\nWe have a short turorial that might help. Hit Repeat tour to begin.`,
+        target: '.trading-hub-header__tradinghub--onboarding--logo',
+        disableBeacon: true,
+    },
+];
+
+export const tour_styles: Styles = {
```

Comment:
Change request: Use our own internal components for Texts and Buttons.

Reply:
and use localize for title and content